scheme/deltas/068-asterisk-subscribecontext.sql

```diff
@@ -0,0 +1,6 @@
+-- Since asterisk 13.17.0 subcribe context is already and endpoint field
+-- but field is defined with underscore
+
+ALTER TABLE `ast_ps_endpoints` ADD `subscribe_context` varchar(40) NOT NULL DEFAULT 'default' AFTER `subscribecontext`;
```

Comment:
What about renaming it with CHANGE instead of all this?
```
ALTER TABLE `ast_ps_endpoints` CHANGE `subscribecontext` `subscribe_context` varchar(40) NOT NULL DEFAULT 'default' AFTER `subscribecontext`;
```